src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/d6bc80a1-68a7-4368-a2d3-77845cff072c)

Reply:
not fixed